### PR TITLE
Remove MaxVSize argument; retry softKill with hardKill; remove RSS backwards compatibility

### LINKED ISSUE #8221: Remove MaxVSize code
Since we don't use it, we better get it out of the code.

### PR DESCRIPTION
**Pull request by amaltaro**
Fixes #8221 , #8222 ,  #8345 and #7274 

Summary of changes are:
* completely remove MaxVSize spec attribute and no longer monitor it in the watchdog
* remove code for backwards compatibility for creation/clone of requests
* remove WMAgent exit code 50661
* if SIGUSR2 signal didn't kill the process in the worker node, retry with a SIGTERM in the next cycle (every 5min)
* remove RSS backwards compatibility to work with RSS either in KiB or MiB

### CODE REVIEW

**Comment by amaltaro** [comment]
Kenyi, I'm merging it in order to cut a new testbed tag for tomorrow, but feel free to review and comment on it.

**Comment by amaltaro** [comment]
I think the T0 and Unified are already setting MaxRSS in MB, but just in case @hufnagel @vlimant 
It's going to get deployed in testbed tomorrow.

File: src/python/WMCore/WMRuntime/Monitors/PerformanceMonitor.py [hunk 1]
```diff
@@ -188,29 +184,22 @@ def periodicUpdate(self):
...
-            vsize = int(output[3])
-        else:
-            # ps returns data in kiloBytes, let's make it megaBytes
-            # I'm so confused with these megabytes and mebibytes...
-            rss = int(output[2]) // 1000  # convert it to MiB
-            vsize = int(output[3]) // 1000  # convert it to MiB
+
+        # ps returns data in kiloBytes, let's make it megaBytes
+        # I'm so confused with these megabytes and mebibytes...
+        rss = int(output[2]) // 1000  # convert it to MiB
```
Comment (khurtado) [inline src/python/WMCore/WMRuntime/Monitors/PerformanceMonitor.py]: This is converting to MegaBytes (MB), not MebiBytes (MiB) as indicated in the comment above, so the conversion comment in this line is wrong.

Comment (amaltaro) [reply src/python/WMCore/WMRuntime/Monitors/PerformanceMonitor.py]: Thanks Kenyi! Noted it in my brain for the next time I touch this module.

**Pull request merged**